_posts/2017-06-03-gsoc-students.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Google Summer of Code Students 2017
+---
+
+We are happy to anounce that MDAnalysis is hosting a [GSoC][gsoc] students for
+the [NumFOCUS][numfocus] this year, [Utkarsh Bansal][utkarsh-gsoc].
+
+# Utkarsh Bansal: Port unit tests to pytest
+
+<img
+src="https://avatars2.githubusercontent.com/u/8345336?v=3&s=460"
+title="Utkarsh Bansal" alt="Utkarsh Bansal"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Utkarsh will port our complete unit tests from nosetest to pytest. This is a
```

Comment:
Maybe add links to [nose](http://nose.readthedocs.io) (not "nosetest") and [pytest](https://pytest.org)